marimo/_runtime/patches.py

```diff
@@ -207,3 +210,181 @@ def patch_main_module_context(
         yield module
     finally:
         sys.modules["__main__"] = main
+
+
+def patch_jedi_parameter_completion() -> None:
+    import re
+
+    from docstring_to_markdown import UnknownFormatError, convert
```

Comment:
this may not be installed (like in wasm). it is still optional, but pulled in from pylsp. could we put a try catch or use `if DependencyManager.docstring_to_markdown.has():`